protoc-gen-grpc-gateway/descriptor/types.go

```diff
@@ -451,7 +451,16 @@ var (
 	}
 
 	wellKnownTypeConv = map[string]string{
-		".google.protobuf.Timestamp": "runtime.Timestamp",
-		".google.protobuf.Duration":  "runtime.Duration",
+		".google.protobuf.Timestamp":   "runtime.Timestamp",
+		".google.protobuf.Duration":    "runtime.Duration",
+		".google.protobuf.StringValue": "runtime.StringValue",
```

Comment:
@achew22 @johanbrandhorst These values don't really exist in [runtime](https://github.com/grpc-ecosystem/grpc-gateway/tree/master/runtime) package. 

So this change only helps in generating reverse proxy (gw.go code) files. It's not sufficient to support "adding protobuf wrappers in url template option" (https://github.com/grpc-ecosystem/grpc-gateway/issues/808).